recognition_engine.py

```diff
@@ -61,7 +61,10 @@ def process_image_msae(image, model_paths, w_height, w_width, mode = 'masks'):
         print str(row) + ' / ' + str(img_height)                
         for col in range(0, img_width - w_width - 1, w_width):
             sample = image[row:row+w_height,col:col+w_width]
-            
+
+            # Pre-process TODO: Check all the pipelines to do the same
+            sample = (255. - sample) / 255.
```

Comment:
Because of this, add `__future__ import division` to the top, because I assume you always want __true division__ not __floor division__. https://www.python.org/dev/peps/pep-0238/

Reply:
You're right. Thank you for bringing this up!